test/src/module/protocol/protocolTest.c

```diff
@@ -374,9 +374,9 @@ testRun(void)
         harnessCfgLoadRaw(strLstSize(argList), strLstPtr(argList));
 
         TEST_RESULT_STR_Z(
-            strLstJoin(protocolRemoteParam(protocolStorageTypePg, 2), "|"),
+            strLstJoin(protocolRemoteParam(protocolStorageTypePg, 1), "|"),
```

Comment:
I feel like we should have a test with the protocolRemoteParam where the pg*-local and pg-default is not always 1.

Reply:
remote always has to be 1 so fixed to do that c04d8fbd35fb7a68e48408b9bc0c301143ef132f